metricbeat/etc/fields.yml

```diff
@@ -417,6 +417,72 @@ system:
             was servicing another processor.
             Available only on Unix.
 
+    - name: system-filesystem
+      type: group
+      description: >
+        `system-filesystem` contains local filesystem stats
+      fields:
+        - name: avail
+          type: integer
```

Comment:
A lot of these are marked as `integer` but are marked as `long`s in Topbeat. Looks like they should `long` because their data types are either int64 or uint64.